docs/src/IceNucleationParcel0D.md

```diff
@@ -196,15 +196,12 @@ It follows that
 where:
 - ``N_{act}`` is the number of activated ice particles.
 
-``N_{act}`` can be computed for example from
-  [activated fraction](https://clima.github.io/CloudMicrophysics.jl/dev/IceNucleation/#Activated-fraction-for-deposition-freezing-on-dust) ``f_i``
-```math
-\begin{equation}
-  N_{act} = N_{aer} f_i
-\end{equation}
-```
-where:
-- ``N_{aer}`` is the number of available dust aerosol particles.
+## Deposition Nucleation on dust particles
+There are multiple ways of running deposition nucleation in the parcel.
+  ``"MohlerAF_Deposition"`` will trigger an activated fraction approach from [Mohler2006](@cite).
```

Comment:
MohlerAF_Deposition here and throughout the added text doesn't render properly. The last line has an example that works